src/Components/dashboard/buses/BusDetail.jsx

```diff
@@ -5,8 +5,26 @@ import Photo from "../../../static/images/busbackground.jpg";
 import { Link } from "react-router-dom";
 import AddAPhotoIcon from "@mui/icons-material/AddAPhoto";
 import DashboardLayout from "../../../Layouts/Dashboard";
+import Sidebar from "../sidebar/Sidebar.jsx";
+import DashNavbar from "../dashnavbar/DashNavBar.jsx";
```

Comment:
As we have a new integrated dashboard design, I hope that we don't need this `DashNavbar` still.